src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -525,6 +531,10 @@ export class MeetingManager implements AudioVideoObserver {
       callback(this.meetingStatus);
     });
   };
+
+  private supportsSetSinkId = (): boolean => {
+    return 'setSinkId' in HTMLAudioElement.prototype;
+  }
```

Comment:
Use `DefaultBrowserBehavior`, assigning one to a field in the constructor.

Reply:
We cannot use the `DefaultBrowserBehavior` as the `supportsSetSinkId` was added in 2.0.0 release and we are trying to make this change compatible with both 1.x & 2.x since the peerDependency is still `1.x || 2.x`.

The idea is, we will remove this and use the `DefaultBrowserBehavior` in the merge for components 2.0 upgrade.

Reply:
Gotcha. 

Btw, if you want this branch to support both, you also need to add `await` for `bindAudioElement` in providers/LocalAudioOutputProvider/index.tsx.

Reply:
Oh, and to catch uses of `chooseAudioInputDevice`; those will throw if used in 2.x and won't in 1.x.

Reply:
Yeah, maybe we can add the awaits like we mentioned. You can refer to the sdk migration doc.

Reply:
Wrapped `chooseAudioInputDevice` inside a `try..catch` & added `await` for `bindAudioElement`